src/views/Home/index.tsx

```diff
@@ -1,12 +1,10 @@
 import { Container, Grid, ThemeProvider } from '@mui/material';
 import Filters from 'components/Filters';
-import Header from 'components/Header';
 import ResultList from 'components/ResultList';
 import theme from 'theme';
 
-const Main = () => (
+const Home = () => (
   <ThemeProvider theme={theme}>
```

Comment:
`<ThemeProvider>` need to be placed somewhere higher, because `Header` component doesn't get theme styles.